test/cpp/ext/otel/otel_plugin_test.cc

```diff
@@ -1956,8 +1956,8 @@ TEST_F(OpenTelemetryPluginCallbackMetricsTest,
       data = collector.Stop();
   // Verify that data is incremental without duplications (cached
   // values).
-  EXPECT_EQ(report_count_1, kIterations);
-  EXPECT_EQ(report_count_2, kIterations);
+  EXPECT_GE(report_count_1, kIterations);
```

Comment:
can you dig into why this might be happening?

Reply:
reverted back and change the duration instead